### PR TITLE
QuickStart: Run Llama 3 on Intel GPU using llama.cpp and ollama with IPEX-LLM

### PR DESCRIPTION
**Pull request by rnwang04**
### 1. Why the change?

https://github.com/analytics-zoo/nano/issues/1285

### 2. User API changes

No change.

### 3. Summary of the change 

add a quickstart about running Llama 3 on Intel GPU using llama.cpp and ollama with IPEX-LLM

### 4. How to test?
- [x] Unit test
- [x] Document test: https://ipex-llm-doc.readthedocs.io/en/add_llama3_quickstart/doc/LLM/Quickstart/llama3_llamacpp_ollama_quickstart.html

### CODE REVIEW

File: docs/readthedocs/source/doc/LLM/Quickstart/llama3_llamacpp_ollama_quickstart.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
+#  Run Llama 3 on Intel GPU using llama.cpp and ollama with IPEX-LLM
+
+[ggerganov/llama.cpp](https://github.com/ggerganov/llama.cpp) prvoides fast LLM inference in in pure C++ across a variety of hardware, [ollama/ollama](https://github.com/ollama/ollama) is popular framework designed to build and run language models on a local machine. You can now use the C++ interface of [`ipex-llm`](https://github.com/intel-analytics/ipex-llm) as an accelerated backend for `llama.cpp` and `ollama` running on Intel **GPU** *(e.g., local PC with iGPU, discrete GPU such as Arc, Flex and Max)*.
+
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/LLM/Quickstart/llama3_llamacpp_ollama_quickstart.md:4]: Not needed

File: docs/readthedocs/source/doc/LLM/Quickstart/llama3_llamacpp_ollama_quickstart.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+Now, you can easily run Llama 3 GGUF models on Intel GPU using `llama.cpp` and `Ollama` with IPEX-LLM.
+
+See the demo of running Llama-3-8B-Instruct on Intel Arc GPU using ollama below.
+
+<video src="https://llm-assets.readthedocs.io/en/latest/_images/ollama-llama3-linux-arc.mp4" width="100%" controls></video>
+
+## Quick Start
+This quickstart guide walks you through how to run Llama 3 on Intel GPU using `llama.cpp` / `Ollama` with IPEX-LLM.
+
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/LLM/Quickstart/llama3_llamacpp_ollama_quickstart.md:15]: 1. run llama 3 using llama.cpp
    1.1 Install
    1.2 Download
    1.3 Run
3. run llama 3 using ollama
    1.1 Install
    1.2 Run

File: docs/readthedocs/source/doc/LLM/Quickstart/llama3_llamacpp_ollama_quickstart.md (hunk 1 above)
Comment (rnwang04) [reply docs/readthedocs/source/doc/LLM/Quickstart/llama3_llamacpp_ollama_quickstart.md]: Done, remove it.

File: docs/readthedocs/source/doc/LLM/Quickstart/llama3_llamacpp_ollama_quickstart.md (hunk 2 above)
Comment (rnwang04) [reply docs/readthedocs/source/doc/LLM/Quickstart/llama3_llamacpp_ollama_quickstart.md:13]: Change to this structure.

**Review by jason-dai — APPROVED** [review]
LGTM

**Pull request merged**